### DISCUSSION TITLE
libp2p with actix-web doesn't work

### DISCUSSION THREAD

**Discussion by kyfk** [libp2p/rust-libp2p#2410] [Q&A]
I'm trying to use axtix-web with libp2p.

I've wrote some codes;
https://github.com/kyfk/spacechat/blob/main/spacechat-agent/src/server.rs#L121-L133

So I was trying to debug the codes whether that works, however it seems Mdns coudn't discover other nodes in my LAN.
When I stop HTTPServer in actix-web, it works fine.

Can anyone tell me why it doesn't work?

**Comment by mxinden** [comment]
First thing that comes to my mind: Are you polling both the HTTPServer and the libp2p `Swarm`?
